cns/cns_util.go

```diff
@@ -106,11 +108,12 @@ func dropUnknownCreateSpecElements(c *Client, createSpecList []cnstypes.CnsVolum
 
 // dropUnknownVolumeMetadataUpdateSpecElements helps drop newly added elements in the CnsVolumeMetadataUpdateSpec, which are not known to the prior vSphere releases
 func dropUnknownVolumeMetadataUpdateSpecElements(c *Client, updateSpecList []cnstypes.CnsVolumeMetadataUpdateSpec) []cnstypes.CnsVolumeMetadataUpdateSpec {
+	var updatedUpdateSpecList []cnstypes.CnsVolumeMetadataUpdateSpec
```

Comment:
Can we initialize this array with the length of updateSpecList?

Reply:
Done